jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/api/JKubeEnricherContextTest.java

```diff
@@ -63,9 +72,7 @@ void builder_whenInvoked_shouldConstructJKubeEnricherContext() {
   void getGav_whenInvoked_shouldReturnExpectedGroupArtifactVersion() {
     // Given + When
     JKubeEnricherContext jKubeEnricherContext = JKubeEnricherContext.builder()
```

Comment:
I see this JKubeEnricherContext builder getting repeated in some tests as well. Could you please move it to setUp method too?